openquakeplatform/settings.py

```diff
@@ -14,6 +14,11 @@
 # Standalone flag to differentiate behaviors
 STANDALONE = True
 
+try:
+    EXTERNAL_TOOLS = True if os.environ['EXTERNAL_TOOLS'] == 'True' else False
```

Comment:
Please, use `os.getenv` instead of the 4 lines with try except statements

Reply:
```suggestion
    EXTERNAL_TOOLS = os.getenv('EXTERNAL_TOOLS', False)
```

Reply:
Fixed with this commit: https://github.com/gem/oq-platform-standalone/pull/60/commits/9ca123232d1e15d9464f903f132e59b66ec79f1f